boards/common/nrf52xxxdk/include/periph_conf.h

```diff
@@ -101,10 +101,10 @@ static const i2c_conf_t i2c_config[] = {
     {
         .dev = NRF_TWIM1,
         .scl = 28,
-        .sda = 29
+        .sda = 29,
```

Comment:
it would be better to use P0.27 for scl and P0.28 for sda, at least for nrf52dk. This way I2C is mapped to the Arduino compatible pinout.
Using P0.28 and P0.29 is also conflicting with Arduino Analog pins.

Reply:
>it would be better to use P0.27 for scl and P0.28 for sda

You mean P0.26 & P0.27 right ? Guess this is not harmful.

Reply:
> You mean P0.26 & P0.27 right ?

Yes, sorry

Reply:
Done.